### DISCUSSION TITLE
Why is supabase dashboard showing so many Auth Request?

### DISCUSSION THREAD

**Discussion by pierroo** [supabase/supabase#20381] [Questions] [answered]
![image](https://github.com/supabase/supabase/assets/17619412/381b50f5-6849-4c11-a7e0-0d9844d6db97)

Title says it all; I have no authentication happening from my app whatsoever;
could these be only admin (dashboard) authentication while navigating the dashboard itself?

**Comment by GaryAustin1** [comment, marked as answer]
The dashboard checks the state of auth server.
If you look in the API edge log you should see things like these:
![image](https://github.com/supabase/supabase/assets/54564956/86188033-cfe0-4a43-87c5-bff5e85a404b)
That is from a new instance doing nothing but the dashboard.  They will be done on regularly while the dashboard is in use.

**Reply by alexander-densley** [reply]
i have the same thing happening, so this is completley fine? it doesn't matter if i have 3000+ auth requests happening?

**Reply by GaryAustin1** [reply]
Hard to know.  You don't even specify a time frame...  
The OP was not using Auth and over a 7 day period of using the dashboard a lot, possibly could get that number. 
I used the dashboard for a few minutes off on and on the log shows this..
![image](https://github.com/supabase/supabase/assets/54564956/01979b89-227c-4995-8fa0-056ce269610c)

You can look at the API report and see if they are health checks (dashbaord use) or something else.  Also I show above filtering the API gateway log to see frequency.

If you are running auth then it could get very high.  getUser calls and token refreshes based on your expire times, etc.
